added test post (#155)
logan-anderson/blog-nextjs-tina-tailwind: content/blog/new-post.md

---
draft: false
date: "2022-06-12"
minRead: "4"
title: Test
author: content/authors/logan_anderson.md
description: test
tags:
  - test
---

> I’m excited to share that Tina’s visual editing UX just got a lot better with click-to-edit functionality.

Author: ```suggestion
I'm excited to share that Tina's visual editing UX has significantly improved with the new click-to-edit functionality.
```

> This pops open the Tina sidebar at the right location when you click editable elements on the page. See it in action on [roev.com](https://www.roev.com/) here:

Author: ```suggestion
This feature opens the Tina sidebar at the correct location when you click on editable elements on the page. You can see it in action on [roev.com](https://www.roev.com/) here:
````

 <CloudinaryVideo src="https://res.cloudinary.com/forestry-demo/video/upload/v1684428946/blog-media/click-to-edit-may-2023/click-to-edit-demo" />
 

>  Our goal is to make visual editing the most intuitive editing experience _without_ compromising the developer's experience and code-quality. Click-to-edit is a big UX win and easy to implement on your site.

Author: ```suggestion
Our goal is to provide the most intuitive visual editing experience without compromising the developer's experience and code quality. Click-to-edit is a major UX improvement and can be easily implemented on your site.
```

 
 ## Demo
 

>  If you want to get a feel for the UX, try this demo (no log in required).

Author: ```suggestion
If you want to experience the UX firsthand, try this demo (no login required).
```

>  <Button link="https://quick-edit-demo.vercel.app/admin" label="Try the Demo" />

Author: ```suggestion
<Button link="https://quick-edit-demo.vercel.app/admin" label="Try the Demo" />
```
